### ISSUE TITLE
Pitch trim going to maximum UP

### ISSUE THREAD

**Issue by lechat74** [flybywiresim/a32nx#7846]
### Aircraft Version

Stable

### Build info

```json
{
    "built": "2023-01-08T23:16:53+00:00",
    "ref": "HEAD",
    "sha": "81a8ee3a2e03a24da13cc0bcc4cf3a7d51f188a6",
    "actor": "Saschl <sascharudolf46@gmail.com>",
    "event_name": "manual",
    "pretty_release_name": "stable/v0.9.1"
}
```

### Describe the bug

Good day,
I am using stable version of A320-251N, and steam version of MSFS 2020
I disabled all AI
I am using Saitek X45 joystick

Rapidly after flight start, pitch trim wheel start to rotate up to saturation
It is not possible to decrease pitch.
It is not possible to control plane, it rapidly take off too early and is going to extreme pitch.
I searched similar issue in history, but only one old in 2020 exists, not being systematic.

### Expected behavior

Pitch trim is stable, around center.

### Steps to reproduce

Just start the flight. Problem is systematic whatever version of A32NX is used.

**Comment by lechat74** [comment]
Checking MSFS debug, 
A32NX_FCDC_1_ELEVATOR_TRIM_POSITION is stuck -62, as well as FCDC_2
A32NX_HYD_TRIM_WHEEL_PERCENT is 99

**Comment by donstim** [comment]
Cannot reproduce. Upon loading a flight starting on the runway, the A32NX_HYD_TRIM_WHEEL_PERCENT is 22. Turning the trim wheel with the mouse while on the ground will change this value accordingly, and it will remain at the value it is at last when the  trim wheel is no longer being moved. 

This appears to be something unique to your configuration. I suggest first trying it without any controllers (joysticks) plugged in.

**Comment by lukecologne** [comment]
> Checking MSFS debug, A32NX_FCDC_1_ELEVATOR_TRIM_POSITION is stuck -62, as well as FCDC_2 A32NX_HYD_TRIM_WHEEL_PERCENT is 99

These variables are encoded as A429 values, and thus are not human-readable from the Lvar debug window. You have to decode them first to get their numerical value.

**Comment by lechat74** [comment]
> > Checking MSFS debug, A32NX_FCDC_1_ELEVATOR_TRIM_POSITION is stuck -62, as well as FCDC_2 A32NX_HYD_TRIM_WHEEL_PERCENT is 99
> 
> These variables are encoded as A429 values, and thus are not human-readable from the Lvar debug window. You have to decode them first to get their numerical value.

yes. I shared value for developer to check if in normal range. I think 99% confirm trim is saturated.
As a side question i wonder: why do we simulate ARINC 429 for entertainement simulator ?
I would expect more simplified fied "black box" mode.

**Comment by lechat74** [comment]
> Cannot reproduce. Upon loading a flight starting on the runway, the A32NX_HYD_TRIM_WHEEL_PERCENT is 22. Turning the trim wheel with the mouse while on the ground will change this value accordingly, and it will remain at the value it is at last when the trim wheel is no longer being moved.
> 
> This appears to be something unique to your configuration. I suggest first trying it without any controllers (joysticks) plugged in.

Thanks Donstim for your answer
Yes, this is unique to my configuration and that's the weird thing. I am in Europe , i will try configuring PC in US regionals.
If pitch trim is saturated up, that would mean we have too much weight in nose, it could come from unit mismatch ?
I will try your recommendation. I already tried to remove pitch trim joystick control, but that did not changed anything
The second weird thing is mouse has no control to wheel. Ths installation is quite fresh, i don't have a lot of changes/history in settings. Problem happened first time i used plane.

**Comment by lukecologne** [comment]
> [...] As a side question i wonder: why do we simulate ARINC 429 for entertainement simulator ? I would expect more simplified fied "black box" mode.

Why not? We try to simulate the aircraft systems as realistically as possible, and this is just the best way to do it. Of course we don't 100% follow the transfer protocol, only the general word structure.
To come back to your issue, could you maybe record a screen capture so we can see the exact behaviour? That could help while debugging.

**Comment by lechat74** [comment]
Hello Luke
Of course we can! this is just more effort. this is the pro way to model accurately sampling/resolution and have good modularity. Good you can go to this level of detail. I worked 4 years as developper of test simulator A330/340. We used real IO A429 card to interface with equipements tested, and shared memory for internal interfaces simulation for simulated systems, using real time transfer function to model exchanges.
I tried without joystick, but same issue.
I recorded video of problem, that start immediately after flight is started. At the end of video i try to control pitch wheel with mouse wheel or clicking/draggin up/down without success. 
I though it could be a CG computation issue, but it maybe something is sending permanent UP commands, blocking other commands. I tried different keyboards, mouses. this does not happen with MSFS planes. I am stuck
Can we record simulated interface with pitch wheel?
BR
PS:
Attached also screenshot of debug aircraft weight

https://user-images.githubusercontent.com/126970538/226190144-c2a1d98b-88ab-4e7c-9eb8-5206cf1c8717.mp4

![CG_1](https://user-images.githubusercontent.com/126970538/226191136-4dbd88dd-4a77-4576-8bd6-fc128f6c98a9.jpg)
![CG_2](https://user-images.githubusercontent.com/126970538/226191138-97ead7ca-2198-4311-abc7-7f2f1bce0f6b.jpg)
![CG_3](https://user-images.githubusercontent.com/126970538/226191139-ac056fc2-7d52-4750-8e21-7f3b44e21022.jpg)

**Comment by lukecologne** [comment]
Hm. The ELACs will not command the THS to move on the ground, except for after landing. It seems like the trim stopped moving while you moved the camera (at around 6 seconds in the video). This makes me think that this is a problem with your sim key/axis bindings, since moving the camera blocks inputs. Would you be able to join our discord for better support? Github is not really ideal for this

**Comment by donstim** [comment]
Discord link here: https://discord.gg/flybywire
CG and anything in the debug weight window does not have anything to do with your issue. As Luke intimated, the THS is only moved on the ground by the pilot. Does this issue occur with both green and yellow hydraulics pumps off (or even cold and dark before engine start)? Can you search for and remove all pitch trim bindings and see if the issue is still there?

**Comment by lechat74** [comment]
ok, clear.
I created account in discord, should i create new issue  or chat in this for this topic ? i am not familiar.
I checked already removing all bindings to pitch trim, no joystick also.
From cold and dark, problem does not appear until Elec Pum Yellow circuit is ON. this is this element specifically that trigger problem
Before, or after, no action is possible in pitch wheel, neither by mouse nor if i attach a keyboard key to it.
So it is not Elec pump that block manual adjustement, this pump just trigger rotation to maximum.
Best regards

**Comment by ImenesFBW** [comment]
> I created account in discord, should i create new issue or chat in this for this topic ?

Hey! Navigate to the #a32nx-supprot channel under the TERMINAL category. Quickly reiterate the problem there and post the link to this issue, so that the folks that will try to help you out see what you have tried already.

**Comment by lechat74** [comment]
> > I created account in discord, should i create new issue or chat in this for this topic ?
> 
> Hey! Navigate to the [#a32nx-support](https://discord.com/channels/738864299392630914/785976111875751956) channel under the TERMINAL category. Quickly reiterate the problem there and post the link to this issue, so that the folks that will try to help you out see what you have tried already.

Thanks. I published in discord.

**Comment by janmg** [comment]
Since update to 0.11 I also have the right mouse button causing trim up, but in msfs2020 options , the mouse button does not report to be mapped as input. I use right mouse button to look around and see the trimwheel spin when I press it, the autopilot recovers trim on release. So as a workaround I look quickly

**Comment by InfectedMushroom9** [comment]
> Since update to 0.11 I also have the right mouse button causing trim up, but in msfs2020 options , the mouse button does not report to be mapped as input. I use right mouse button to look around and see the trimwheel spin when I press it, the autopilot recovers trim on release. So as a workaround I look quickly

Can you try changing right mouse button binding from `Toggle Freelook (Hold)` to `Toggle Freelook`?

There is a MSFS bug since at least SU5 that also messes up stick inputs when hand-flying.
